.travis.yml

```diff
@@ -2,13 +2,15 @@ language: python
 
 matrix:
  include:
-  - python: "2.7"
+  - python: "3.5"
     env: DISTRIBUTIONS="sdist bdist_wheel --universal"
   - python: "3.6"
   - python: "3.7"
     dist: latest  # required for Python >= 3.7 (travis-ci/travis-ci#9069)
-  - python: "2.7"
+  - python: "3.5"
     env: TOXARG="-e pylint"
+  - python: "3.6"
+    env: TOXARG="-e black-test"
```

Comment:
black not working under Python 3.5 is going to be annoying given that we'll probably be running 3.5 as our default dev environment in most repos for another few months.

Reply:
Yea, I think it may point to us waiting to actually deploy this until we're on 3.6 and above.  Especially since python 3.6 won't be available on devstack by default.  I think this was a good learning experience but we may want to hold off on actually turning on black for everything until we're on a newer version of python.  @nedbat @jmbowman thoughts?